ANSWERS.md

```diff
@@ -1,4 +1,8 @@
-What is the DOM? The Document Object Model is a interface for programming with HTML and related documents that allows for manipulation and giving functionality to elements on the page. It is a bringing together of HTML(for the base of the application), CSS (for the styling), and finally really working in the JavaScript level of this triad to bring action to Web Design. DOM is manipulating elements in JavaScript and affecting them up through this triad of related files. 
-What is an event? An event is an action on a Web Page initiated by the user. Javascript allows a small program function to occur when called. For example, a hover, a click, a mouse hold or release, or a specific button click can initiate an event or series of events in Web Design. 
-What is an event listener? An event listener is a specific syntax tag added to a callback function (usually in the JS file) that allows a specific element on a web page to be responsive to the user. Example: this.element.addEventListener('click', runFile); would call the 'runFile' function when the target element was clicked. 
-What is a Constructor? A constructor is what is used for an 'instance method' in object orientated programming. It is the parent in a Class structure that initializes objects of that type. For manipulating elements in JS the first line 'this.element = element' is used.   
\ No newline at end of file
+##What is the DOM?## 
+The Document Object Model is a interface for programming with HTML and related documents that allows for manipulation and giving functionality to elements on the page. It is a bringing together of HTML(for the base of the application), CSS (for the styling), and finally really working in the JavaScript level of this triad to bring action to Web Design. DOM is manipulating elements in JavaScript and affecting them up through this triad of related files. 
```

Comment:
Awesome answers! I would only say that your last sentence is a little off. `DOM *Manipulation* is manipulating elements ....`. You just forgot to add the manipulation 😄